src/devices/bus/gameboy/mbc.cpp

```diff
@@ -155,6 +156,15 @@ gb_rom_camera_device::gb_rom_camera_device(const machine_config &mconfig, const
 {
 }
 
+gb_rom_vfame_device::gb_rom_vfame_device(const machine_config &mconfig, device_type type, const char *tag, device_t *owner, uint32_t clock)
+	: gb_rom_mbc5_device(mconfig, type, tag, owner, clock)
+{
+}
```

Comment:
This protected constructor is unnecessary since there are no subclasses of `gb_rom_vfame_device`. Compare with how most of the other cart types only have one constructor.